projects/valtimo/dossier/src/lib/components/dossier-detail/tab/widgets/components/field/widget-field.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<ng-container
+  *ngIf="{
+    widgetConfiguration: widgetConfiguration$ | async,
+    widgetPropertyValue: widgetPropertyValue$ | async
+  } as obs"
+>
+  <h4 class="widget-field__title">{{ obs?.widgetConfiguration?.title }}</h4>
+
+  <div class="widget-field__container">
+    @for (column of obs?.widgetPropertyValue; track column.key) {
+      <div class="widget-field__column">
+        @for (property of column; track property) {
+          <cds-label>
+            {{ property?.title }}
+            <input cdsText [value]="property?.value" [readonly]="true" />
```

Comment:
You have not taken into account displayProperties on value (currency etc should be formatted). Contact @teodor-ritense on how to best implement this